Makefile

```diff
@@ -29,18 +29,20 @@ build: test-data
 	mvn --batch-mode --update-snapshots package
 
 ## test-data
-testDataDir := src/test/resources/
-tempDir := ${testDataDir}temp/
-gitDataDir := ${tempDir}sdk-test-data/
+testDataDir := src/test/resources
+banditsDataDir := ${testDataDir}/bandits
```

Comment:
I modified this to leave bandit test files that have been checked into the repository alone. This will no longer be an issue once we merge with the common test repository.